tools/rpmuncompress.c

```diff
@@ -22,6 +26,8 @@ static struct poptOption optionsTable[] = {
 	N_("provide more detailed output"), NULL },
     { "dry-run", 'n', POPT_ARG_VAL, &dryrun, 1,
 	N_("only print what would be done"), NULL },
+    { "path", 'C', POPT_ARG_STRING, &dstpath, 0,
+	N_("extract into a specific path"), NULL },
```

Comment:
Hmm, -C makes sense in the %setup where it relates to -c, but as a shortcut for --path it seems strange.

Reply:
Heh, but then `tar` has -C for --directory for a kind of similar operation.

Reply:
what do you suggest for short and long argument?
Should we keep %autosetup and %setup consistent?

Reply:
It's no big deal. We can always blame tar for precedence :smile:

Reply:
Interestingly, tar has an option to do this without having to create the directory manually: `--one-top-level` [was introduced in 2014](https://git.savannah.gnu.org/cgit/tar.git/commit/?id=2af87fa2776c8125a587a9b0c2c4fae3bf921ff7).
But I didn't use it because it won't work with other implementations like BSD or Mac.